Gems/ROS2/Code/Source/GNSS/GNSSFormatConversions.cpp

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include "GNSSFormatConversions.h"
+
+constexpr double earthSemimajorAxis = 6378137.0f;
+constexpr double reciprocalFlattening = 1 / 298.257223563f;
+constexpr double earthSemiminorAxis = earthSemimajorAxis * (1.0f - reciprocalFlattening);
+constexpr double firstEccentricitySquared = 2 * reciprocalFlattening - reciprocalFlattening * reciprocalFlattening;
+constexpr double secondEccentrictySquared =
+    reciprocalFlattening * (2.0f - reciprocalFlattening) / ((1.0f - reciprocalFlattening) * (1.0f - reciprocalFlattening));
+
+// Based on http://wiki.gis.com/wiki/index.php/Geodetic_system
+namespace ROS2::GNSS
+{
+    float Rad2Deg(float rad)
+    {
+        return rad * 180.0f / AZ::Constants::Pi;
+    }
+
+    float Deg2Rad(float deg)
+    {
+        return deg * AZ::Constants::Pi / 180.0f;
+    }
```

Comment:
Instead of defining new functions for this, the existing RadToDeg and DegToRad functions can be used